skyplane/cli/cli.py

```diff
@@ -136,16 +138,43 @@ def cp(
 
     requester_pays: bool = cloud_config.get_flag("requester_pays")
 
-    if provider_src == "local" or provider_dst == "local":
-        typer.secho("Local transfers are not yet supported (but will be soon!)", fg="red", err=True)
-        typer.secho("Skyplane is currently most optimized for cloud to cloud transfers.", fg="yellow", err=True)
-        typer.secho(
-            "Please provide feedback for on prem transfers at: https://github.com/skyplane-project/skyplane/discussions/424",
-            fg="yellow",
-            err=True,
+    if provider_src == "local" and provider_dst == "local":
+        typer.secho("Copying between local paths", fg="yellow")
+        process = subprocess.Popen(["cp", "-r", path_src, path_dst], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        for line in iter(process.stdout.readline, b""):
+            typer.secho(f"{line}\n", fg="yellow")
+        for line in iter(process.stdout.readline, b""):
+            typer.secho(f"Error: {line}\n", fg="red")
+
+    elif provider_src == "local" or provider_dst in clouds:
+        typer.secho(f"On-prem to {provider_dst} transfer. Defaulting to native {provider_dst} offering", fg="yellow")
+        process = subprocess.Popen(
+            cloud_provider_api[provider_dst] + [path_src, f"s3://{bucket_dst}/{path_dst}"] + ["--recursive"]
+            if provider_dst == "s3"
+            else [],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
         )
```

Comment:
Use os.system to run commands which will automatically forward stdout and stderr.